frontend/src/entities/dataset/detail/dataset-detail.tsx

```diff
@@ -68,15 +68,15 @@ function DatasetDetail () {
             setBreadcrumbs([
                 getBase(projectName),
                 { text: 'Datasets', href: `#${basePath}/dataset` },
-                { text: `${name}`, href: `#${basePath}/dataset/${scope}/${name}` },
+                { text: `${name}`, href: `#${basePath}/dataset/${type}/${scope}/${name}` },
             ])
         );
-        dispatch(getDatasetByScopeAndName({ scope: scope, name: name }))
+        dispatch(getDatasetByScopeAndName({ type: type, scope: scope, name: name }))
```

Comment:
`getDatasetsByScopeAndName` probably isn't a good name anymore :x

Reply:
`getDatasetByScope,Name,andFriends` is my vote

Reply:
`getDataset` seems available, so I'll go with that for brevity.